FDK17プロジェクト/コード/03.サウンド/ESoundGroup.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace FDK
+{
+    public enum ESoundGroup
```

Comment:
Not much to say about ESoundGroup that you wouldn't have already figured out. This is the first place to go if one wants to add new sound group types and distribute existing sounds across them so that there can be even more independent volume controls for new groups.